manifests/a/ArmCord/ArmCord/3.2.0/ArmCord.ArmCord.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with WinGet Releaser v2 using Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ArmCord.ArmCord
+PackageVersion: 3.2.0
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-05-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ArmCord/ArmCord/releases/download/v3.2.0/ArmCord.Setup.3.2.0.ARM64.exe
+  InstallerSha256: 8AB84F27C7284E0FC55E549CED81DD9B8AE2EB9FD204DC44614835929C7DE9DE

```

Comment:
Incorrect Url is used. Tracked in https://github.com/russellbanks/Komac/issues/200.

Manually fixed in #107190 for now.